packages/mdc-select/_variables.scss

```diff
@@ -24,7 +24,7 @@
 
 $mdc-select-arrow-padding: 26px;
 $mdc-select-label-padding: 16px;
-$mdc-select-border-radius: 4px;
+$mdc-shape-height: 56px;
```

Comment:
I assume this should be mdc-select height, not mdc-shape-height. Also, is this value really the same for both variants?

Reply:
Yes, corrected.